app/(dashboard)/public-cloud/products/active-requests/page.tsx

```diff
@@ -1,68 +1,61 @@
 import Table from '@/components/table/Table';
-import TableBody from '@/components/table/TableBodyProducts';
-import { publicCloudRequestsPaginated } from '@/queries/paginated/public-cloud';
-import { PublicProject } from '@/queries/types';
+import NewTableBody from '@/components/table/TableBodyProducts';
 import { publicCloudProjectDataToRow } from '@/components/table/helpers/row-mapper';
 import { getServerSession } from 'next-auth/next';
 import { authOptions } from '@/core/auth-options';
 import { redirect } from 'next/navigation';
-import { userInfo } from '@/queries/user';
+import { parsePaginationParams } from '@/helpers/pagination';
+import { searchActivePublicCloudRequests } from '@/queries/public-cloud-requests';
 
 export default async function ProductsTable({
   searchParams,
 }: {
   searchParams: {
     search: string;
-    page: number;
-    pageSize: number;
+    page: string;
+    pageSize: string;
     ministry: string;
     provider: string;
   };
 }) {
-  // Authenticate the user
   const session = await getServerSession(authOptions);
 
   if (!session) {
-    redirect('/login?callbackUrl=/private-cloud/products/all');
+    redirect('/login?callbackUrl=/public-cloud/products/all');
   }
 
-  const { search, page, pageSize, ministry, provider } = searchParams;
-  const { userEmail, ministryRoles } = userInfo(session.user.email, session.roles);
+  const { search, page: pageStr, pageSize: pageSizeStr, ministry, provider } = searchParams;
 
-  // If a page is not provided, default to 1
-  const currentPage = typeof searchParams.page === 'string' ? +page : 1;
-  const defaultPageSize = 10;
+  const { page, skip, take } = parsePaginationParams(pageStr, pageSizeStr, 10);
 
-  const effectivePageSize = +pageSize || defaultPageSize;
-
-  const { data: requestsData, total: requestsTotal } = await publicCloudRequestsPaginated(
-    effectivePageSize,
-    currentPage,
-    search,
+  const { docs, totalCount } = await searchActivePublicCloudRequests({
+    session,
+    skip,
+    take,
     ministry,
     provider,
-    userEmail,
-    ministryRoles,
-  );
+    search,
+  });
 
-  const transformActiveRequests = requestsData.map((request) => ({
-    ...request.requestedProject,
+  const transformActiveRequests = docs.map((request) => ({
+    ...request.userRequestedProject,
     created: request.created,
     updatedAt: request.updatedAt,
-    activeRequest: [request],
+    requests: [request],
     id: request.id,
   }));
 
   const activeRequests = transformActiveRequests.map(publicCloudProjectDataToRow);
 
   return (
     <Table
-      title="Products in Public Cloud Landing Zones"
+      title="Products in Public Cloud OpenShift Platform"
```

Comment:
I don't think this is supposed to read "Public Cloud OpenShift Platform" either `"Public Cloud Landing Zone"` or `"Private Cloud OpenShift Platform"`